src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishReleaseDataFunction.cs

```diff
@@ -50,9 +55,13 @@ public class PublishReleaseDataFunction
         {
             logger.LogInformation($"{executionContext.FunctionName} triggered: {message}");
 
-            if (PublisherUtils.IsDevelopment())
+            var releaseHasSubjects = _statisticsDbContext
+                .ReleaseSubject
+                .Where(rs => rs.ReleaseId == message.ReleaseId)
+                .Any();
+            if (PublisherUtils.IsDevelopment() || !releaseHasSubjects)
```

Comment:
You could consider making releaseHasSubjects a private method to keep this a bit shorter to read